catalog/core/catalog-core-camelcomponent/src/main/java/ddf/camel/component/catalog/CatalogEndpoint.java

```diff
@@ -204,4 +209,26 @@ public boolean isSingleton() {
   public boolean isMultipleConsumersSupported() {
     return true;
   }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (!(o instanceof CatalogEndpoint)) {
+      return false;
+    }
+    if (!super.equals(o)) {
+      return false;
+    }
+    CatalogEndpoint that = (CatalogEndpoint) o;
+    return transformerId.equals(that.getTransformerId())
+        && contextPath.equals(that.getContextPath())
+        && mimeType.equals(that.getMimeType());
```

Comment:
❓ Are these the right fields to include?

Reply:
That's what I would have done 👍